gammapy/irf/irf_stack.py

```diff
@@ -0,0 +1,94 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+import numpy as np
+from astropy.units import Quantity
+from ..irf import EffectiveAreaTable, EnergyDispersion
+
+__all__ = [
+    'IRFstack',
+]
+
+log = logging.getLogger(__name__)
+
+
+class IRFstack(object):
+    """
+    Compute the mean effective area and the mean rmf for several observations for a given list of rmf and arf
+
+    Parameters
+    ----------
+    list_arf: list
+        list of arf
```

Comment:
It's clearer what the user has to input if you write 
```
list_arf : list
    list of `~gammapy.irf.EffectiveAreaTable`
```
the same hold for the other input parameters. In the documentation this will generate a link to the correct class. see for example: http://docs.gammapy.org/en/latest/api/gammapy.spectrum.SpectrumObservation.html#gammapy.spectrum.SpectrumObservation